srs_data/splitting.py

```diff
@@ -0,0 +1,81 @@
+import os
+import subprocess
+import sys
+
+from constants import ASAPP_ROOT, ASAPP_MLENG_ROOT
+from constants import CLIENT_FULL_NAMES
+from base import BaseTool
+
+import pandas as pd
+
+
+class SplitProcessedTagsIntoDataCorpora(BaseTool):
+
+    EXPECTED_INPUT_FILE_COLUMNS = ['text', 'tag', 'notes']
+    SUFFIX_REPLACEMENT_LOOKUP = {
+        '_0': 'a',
+        '_1': 'b',
+        '_2': 'c'
```

Comment:
instead of replacing these suffixes, let's just name them directly with 'a', 'b', 'c' when generated